src/st/entity_axe_knight.h

```diff
@@ -1,37 +1,67 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 
+// Code in this file is very similar to e_blue_axe_knight.h
+// minus some additional attacks and code not present in the higher
+// level blue axe knight
+
 typedef enum {
     AXE_KNIGHT_INIT,
     AXE_KNIGHT_IDLE,
     AXE_KNIGHT_WALK_TOWARDS_PLAYER,
     AXE_KNIGHT_WALK_AWAY_FROM_PLAYER,
     AXE_KNIGHT_STANDING_THROW,
     AXE_KNIGHT_DUCKING_THROW,
-    AXE_KNIGHT_UNUSED, // Charge Attack missing step from the blue AxeKnight
-    AXE_KNIGHT_ARCING_THROW, // Unused, present in the blue AxeKnight
+    AXE_KNIGHT_CHARGE_ATTACK, // Unused, present in the blue AxeKnight
+    AXE_KNIGHT_ARCING_THROW,  // Unused, present in the blue AxeKnight
     AXE_KNIGHT_DYING,
 } EntityAxeKnightSteps;
 
 static s16 sensors_move[] = {0, 32, 8, 0};
 static s16 sensors_ground[4][2] = {{0, 32}, {0, 4}, {8, -4}, {-16, 0}};
 static s16 dead_particle_pos[][2] = {
     {-8, -4}, {8, -6}, {-2, 4}, {-1, -7}, {2, 8}, {11, -7}, {4, 6}, {-3, 0}};
-static u16 unused[] = {
-    0x0114, 0x020A, 0x030A, 0x0414, 0x030A, 0x020A, 0x0000, 0x0000, 0x0508,
-    0x0608, 0x0708, 0x0D08, 0x0E08, 0x0F02, 0x1402, 0x1502, 0x1620, 0x1708,
-    0x0000, 0x0000, 0x0508, 0x0608, 0x0708, 0x0808, 0x090A, 0x0A04, 0x0B04,
-    0x0C04, 0x0000, 0x0000, 0x0E04, 0x0F04, 0x1004, 0x1104, 0x0000, 0x0000,
-    0x0D08, 0x0E08, 0x0F01, 0x1001, 0x1101, 0x1220, 0x1308, 0x0000};
```

Comment:
While decompiling the blue axe knight I found this is actually 5 separate animations. I've moved those changes over here too for completeness, even though these aren't used